numpy/_core/src/multiarray/einsum.c.src

```diff
@@ -1035,7 +1035,6 @@ PyArray_EinsteinSum(char *subscripts, npy_intp nop,
     iter_flags = NPY_ITER_EXTERNAL_LOOP|
             NPY_ITER_BUFFERED|
             NPY_ITER_DELAY_BUFALLOC|
-            NPY_ITER_GROWINNER|
```

Comment:
How about adding a comment here? Partially as a reminder that a pair-wise sum might be a good idea...

Reply:
Yeah, didn't think of the "let's do pairwise" spin, so wasn't sure.